sympy/functions/combinatorial/numbers.py

```diff
@@ -2175,6 +2175,149 @@ def eval(cls, n):
         return Integer(cls._motzkin(n - 1))
 
 
+#-----------------------------------------------------------------------------#
+#                                                                             #
+#                         Schoeder numbers                                    #
+#                                                                             #
+#-----------------------------------------------------------------------------#
+
+
+class schroeder(Function):
+    r"""
+    In mathematics, the Schroeder number,
+    also called a large Schroeder number or
+    big Schroeder number, describes the number of
+    lattice paths from the southwest corner $(0, 0)$
+    of an $n\times n$ grid to the northeast corner $(n, n)$ using only
+    single steps north, $(0, 1)$ northeast, $(1, 1)$ or east, $(1, 0)$
+    that do not rise above the SW-NE diagonal. The nth
+    Schroeder number is the number of Schroeder paths
+    of length n. The Schroeder numbers  were named
+    after the German mathematician Ernst Schroeder.
```

Comment:
Any decent code editor will be able to fix this for you by wrapping lines to a fixed length.

Reply:
I have formatted the docstring such that each line has length of only 72 characters. Is it the correct way?